docs/src/build-from-source.md

```diff
@@ -29,15 +29,15 @@ cd kani
 git submodule update --init
 ./scripts/setup/ubuntu/install_deps.sh
 ./scripts/setup/ubuntu/install_cbmc.sh
-./scripts/setup/install_viewer.sh 3.5
-./scripts/setup/install_rustup.sh
+./scripts/setup/install_viewer.sh
 # If you haven't already:
+./scripts/setup/install_rustup.sh
```

Comment:
nit: `install_rustup.sh` already includes the next line that sources `~/.cargo/env`.

Reply:
That's a bug in the script, because `source` would only affect the rest of that script, not the user's shell.

Reply:
Ah, indeed!